docs/tutorials/python/profiler.md

```diff
@@ -212,11 +212,6 @@ Should the existing NDArray operators fail to meet all your model's needs, MXNet
 Let's try profiling custom operators with the following code example:
 
 ```python
-
-import mxnet as mx
```

Comment:
why remove this?

Reply:
They are imported in the code above. I though maybe I should avoid duplication